services/web/server/src/simcore_service_webserver/storage_api.py

```diff
@@ -54,31 +59,38 @@ async def get_storage_locations(
         return locations_enveloped.data
 
 
-async def get_project_total_size(
+async def get_project_total_size_simcore_s3(
     app: web.Application, user_id: UserID, project_uuid: ProjectID
 ) -> ByteSize:
-    log.debug("getting %s total size for %s", f"{project_uuid=}", f"{user_id=}")
-    user_accessible_locations = await get_storage_locations(app, user_id)
-    session, api_endpoint = _get_storage_client(app)
-
-    project_size_bytes = 0
-    for location in user_accessible_locations:
-        files_metadata_url = (
-            api_endpoint / "locations" / f"{location.id}" / "files" / "metadata"
-        ).with_query(user_id=user_id, uuid_filter=f"{project_uuid}")
-        async with session.get(f"{files_metadata_url}") as response:
-            response.raise_for_status()
-            list_of_files_enveloped = Envelope[list[FileMetaDataGet]].parse_obj(
-                await response.json()
-            )
-            assert list_of_files_enveloped.data is not None  # nosec
-        for file_metadata in list_of_files_enveloped.data:
-            project_size_bytes += file_metadata.file_size
-    project_size = parse_obj_as(ByteSize, project_size_bytes)
-    log.info(
-        "%s total size is %s", f"{project_uuid}", f"{project_size.human_readable()}"
-    )
-    return project_size
+    with log_context(
+        log,
+        logging.DEBUG,
+        msg=f"getting {project_uuid=} total size in S3 for {user_id=}",
+    ):
+        # NOTE: datcore does not handle filtering and is too slow for this, so for now this is hard-coded
+        user_accessible_locations = [FileLocation(name="simcore.s3", id=0)]
+        session, api_endpoint = _get_storage_client(app)
+
+        project_size_bytes = 0
+        for location in user_accessible_locations:
+            files_metadata_url = (
+                api_endpoint / "locations" / f"{location.id}" / "files" / "metadata"
+            ).with_query(user_id=user_id, uuid_filter=f"{project_uuid}")
+            async with session.get(f"{files_metadata_url}") as response:
+                response.raise_for_status()
+                list_of_files_enveloped = Envelope[list[FileMetaDataGet]].parse_obj(
+                    await response.json()
+                )
+                assert list_of_files_enveloped.data is not None  # nosec
+            for file_metadata in list_of_files_enveloped.data:
+                project_size_bytes += file_metadata.file_size
+        project_size = parse_obj_as(ByteSize, project_size_bytes)
+        log.info(
```

Comment:
MINOR: i know this was there before but just as a comment. I do not see the point of logging.INFO a return value. It should be the caller who does that (if needed) since he has both the ``project_uuid`` (input) and ``project_zsiez`` (output). 
For the sake of discussion: imaging we call this function in a for loop to find some specific size ...  you get an unwished info log.

Reply:
we are anyway not logging info logs

Reply:
I remove it. But then I can ask for a lot of the logs... imagine we have 10000 users calling into the handlers...

Reply:
i am not saying you should not put logs. I am saying that IMO it should be placed outside this function i.e. by the caller